### PR TITLE
feat(compute): Added example for mig standby policy

### PR DESCRIPTION
**Pull request by askubis**
## Description

Fixes #335382497

Note: If you are not associated with Google, open an issue for discussion before submitting a pull request.

## Checklist

**Readiness**

- [x] Yes, **merge** this PR after it is approved
- [ ] No, don't **merge** this PR after it is approved

**Style**

- [x] My sample follows the rules described for Terraform in the [Effective Samples style guide](https://googlecloudplatform.github.io/samples-style-guide/)
- [x] My sample follows the other requirements and best practices in the [Contributing
guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#other-requirements-and-best-practices)

**Testing**

- [x] I have performed tests described in the [Contributing guide](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md):

   - [x] **[Tests](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#set-up-the-test-environment)** pass: `terraform apply`
   - [x] **[Lint](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/CONTRIBUTING.md#linting-and-formatting)** pass: `terraform fmt` check

**Intended location**

- [x] Yes, this sample will be (or already is) included on cloud.google.com
      Location(s): https://cloud.google.com/compute/docs/instance-groups/accelerate-mig-scale-out-with-standby-pools#edit_the_standby_policy_in_a_mig

- [ ] No, this sample won't be included on cloud.google.com
      Reason:

**API enablement**

- [x] If the sample needs an API enabled to pass testing, I have added the service to the [Test setup file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/test/setup/main.tf)

**Review**

- [x] If this sample adds a new directory, I have added codeowners to the [CODEOWNERS file](https://github.com/terraform-google-modules/terraform-docs-samples/blob/main/.github/CODEOWNERS)

### CODE REVIEW

File: compute/zonal_mig_standby_policy/main.tf [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  name               = "standby-mig"
+  base_instance_name = "test"
+  target_size        = 3
+  zone               = "us-central1-f"
+
+  version {
+    instance_template = google_compute_instance_template.default.id
+    name              = "primary"
+  }
+  standby_policy {
```
[the remark this answers is not in the document]
Comment (msampathkumar) [reply compute/zonal_mig_standby_policy/main.tf:48]: CC: @askubis

[the remark this answers is not in the document]
Comment (msampathkumar) [reply compute/zonal_mig_standby_policy/main.tf:48]: Update: If CICD test passes, I am inclined to approve this PR. But looking at long term perspective, I reached out to terraform-chat group for more details on this.

[the remark this answers is not in the document]
Comment (rileykarson) [reply compute/zonal_mig_standby_policy/main.tf:48]: That's unmanaged instance group, you want managed instance group! https://registry.terraform.io/providers/hashicorp/google-beta/latest/docs/resources/compute_instance_group_manager#standby_policy

**Pull request merged**